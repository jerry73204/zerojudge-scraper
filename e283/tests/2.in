14
0 1 0 1
0 1 1 1
1 1 0 1
0 0 1 0
0 0 1 0
1 1 0 0
0 1 0 1
1 0 0 0
0 1 1 1
0 0 1 0
1 0 0 0
0 0 1 0
0 1 1 1
0 0 1 0
17
1 0 0 0
0 1 0 1
1 0 0 0
1 1 0 0
0 1 1 1
0 0 1 0
1 1 0 0
0 0 1 0
0 1 0 1
1 1 0 1
0 1 0 1
0 1 0 1
1 0 0 0
1 1 0 0
0 1 0 1
0 0 1 0
0 0 1 0
2
1 1 0 0
1 1 0 0
17
1 1 0 1
1 0 0 0
1 1 0 0
0 1 1 1
0 0 1 0
1 1 0 0
0 0 1 0
0 0 1 0
1 0 0 0
0 1 0 1
0 1 0 1
0 0 1 0
1 1 0 1
1 0 0 0
1 1 0 1
0 0 1 0
1 0 0 0
10
1 1 0 1
1 0 0 0
1 1 0 0
1 1 0 1
0 1 0 1
1 1 0 1
1 0 0 0
1 1 0 1
0 1 0 1
1 0 0 0
14
0 1 0 1
0 1 1 1
1 1 0 0
0 0 1 0
0 1 0 1
1 1 0 1
1 0 0 0
1 1 0 1
0 1 1 1
1 0 0 0
1 1 0 1
1 1 0 0
0 1 1 1
1 0 0 0
18
0 1 0 1
1 0 0 0
0 0 1 0
1 0 0 0
1 1 0 1
1 1 0 0
1 0 0 0
0 1 0 1
1 1 0 0
1 1 0 0
0 1 1 1
1 1 0 1
1 1 0 1
1 1 0 1
0 1 1 1
1 1 0 0
1 0 0 0
0 0 1 0
2
0 0 1 0
0 1 0 1
18
0 0 1 0
0 0 1 0
1 1 0 1
1 0 0 0
1 0 0 0
0 1 0 1
0 0 1 0
1 0 0 0
0 1 0 1
0 0 1 0
0 0 1 0
1 0 0 0
0 1 0 1
1 0 0 0
1 1 0 1
0 0 1 0
0 0 1 0
1 1 0 0
9
0 1 1 1
1 1 0 1
0 0 1 0
0 1 0 1
0 1 1 1
0 0 1 0
1 0 0 0
0 1 1 1
0 0 1 0
