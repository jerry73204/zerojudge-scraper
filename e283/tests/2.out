ABDCCFAEBCECBC
EAEFBCFCADAAEFACC
FF
DEFBCFCCEAACDEDCE
DEFDADEDAE
ABFCADEDBEDFBE
AECEDFEAFFBDDDBFEC
CA
CCDEEACEACCEAEDCCF
BDCABCEBC